### PR TITLE
[DH-276] update cluster switchover doc

### PR DESCRIPTION
**Pull request by shaneknapp**
update our cluster switchover docs to something relevant, recent and (mostly) complete.

at some point, i'd love for this doc to be used as a template for a "how to deploy our infra from zero to datahub".

### CODE REVIEW

File: docs/admins/howto/clusterswitch.md [hunk 1]
```diff
@@ -4,11 +4,87 @@ This document describes how to switch an existing hub to a new cluster.  The exa
...
+		kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${cert-manager-version}/cert-manager.yaml
+
+4. Change the corresponding entry in `support/requirements.yaml` to `$cert-manager-version` and commit the changes (do not push).
+
+## Create the node-placeholder k8s namespace
+The [calendar autoscaler](https://docs.datahub.berkeley.edu/en/latest/admins/howto/calendar-scaler.html) requires the `node-placeholder` namespace.  Run the following command to create it:
+
+		kubectl create namespace node-placeholder
+
+## Switch DNS to the new cluster's endpoint IP and point our deployment at it.
```
Comment (shaneknapp) [inline docs/admins/howto/clusterswitch.md]: @felder i could definitely use your eyes on this section...  :)

**Comment by shaneknapp** [comment]
@ryanlovett @felder it'd be great if you could put your peepers on these changes for any feedback.

thanks!

**Comment by ryanlovett** [comment]
@shaneknapp Seems clear to me! I didn't participate in most the upgrade so I don't know if anything is missing. The cert-manager section seems accurate though, from what I recall.

Either in this PR or another, it might be good to document in a sentence or two why it can be necessary to deploy a new cluster.

File: docs/admins/howto/clusterswitch.md (hunk 1 above)
Comment (felder) [reply docs/admins/howto/clusterswitch.md]: Yeah I think this is not quite right.

I don't think the endpoint is relevant.  So basically what infoblox points to is the loadbalancer that's created as part of the ingress-controller service in the support chart.  So roughly, I believe the series of steps for this is to:

1.  Allocate a new static ip via the GCP console.
2. Configure the wildcard and empty (datahub.berkeley.edu) records in infoblox to point to the new ip.
3. Configure the loadBalancerIP for ingress-nginx in support/values.yaml to use the new ip.
4. Redeploy the support chart to pick up the change.
5. Confirm via the GCP console that the IP that was defined in step 1 is now bound to a forwarding rule.  You can further confirm by listing the services in the support chart and making sure the ingress-controller is using the newly defined IP.

**Review by felder — DISMISSED** [review]
I commented and suggested edits for the specific section you asked me about.  I think everything else looks reasonable, but it's hard to know for sure.  I guess we can cross  that bridge the next time we do a cluster rotation and refer to this doc.

**Comment by shaneknapp** [comment]
@felder let me know if the changes from [79bcd14f1f76242984e362c92bffabb9ebb66261](https://github.com/berkeley-dsep-infra/datahub/pull/5750/commits/79bcd14f1f76242984e362c92bffabb9ebb66261) look good!

**Comment by shaneknapp** [comment]
> @shaneknapp Seems clear to me! I didn't participate in most the upgrade so I don't know if anything is missing. The cert-manager section seems accurate though, from what I recall.
> 
> Either in this PR or another, it might be good to document in a sentence or two why it can be necessary to deploy a new cluster.

https://github.com/berkeley-dsep-infra/datahub/pull/5750/commits/0558bb96f43f4fe25308d2af4f48b102c4735046
and
https://github.com/berkeley-dsep-infra/datahub/pull/5750/commits/a788ff38d3fcc878d4ad994c8536566fb56e46fd

**Pull request merged**